src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected LocalDate deadlineFrom;
+    protected LocalDate deadlineTo;
+    String DATE_FORMAT = "MMM d yyyy";
+    String DIVIDER = " | ";
```

Comment:
Should this be made `static final`? I noticed the same issue in several other places too. Press on! 👍